recipes/protobuf/all/conanfile.py

```diff
@@ -58,23 +79,32 @@ def config_options(self):
     def configure(self):
         if self.options.shared:
             self.options.rm_safe("fPIC")
+            if Version(self.version) >= "3.22.0" and is_msvc(self):
+                self.options["abseil"].shared = True
```

Comment:
Please, point a reference why is it needed.

Reply:
It was added by @toge, so I had to look it up. Appears to be related to https://github.com/protocolbuffers/protobuf/blob/v25.3/cmake/abseil-cpp.cmake#L40-L54

I would leave it to a follow-up PR to fix this unless you can suggest a simple fix/workaround to this.

Reply:
Failed with
```
LINK : fatal error LNK1104: cannot open file 'abseil_dll.lib' [C:\J\w\prod\BuildSingleReference@3\s\24cac7\1\build\libprotobuf.vcxproj]
```
in the CI before this change.